djconnectwise/sync.py

```diff
@@ -93,6 +93,7 @@ class Synchronizer:
 
     def __init__(self, full=False, *args, **kwargs):
         self.api_conditions = []
+        self.partial_sync_support = True
```

Comment:
Can I ask for your explanation for an introduction of `partial_sync_support`? Until now, we support partial & full sync wo/ any other value, so why is that added here?

Reply:
I had to add this since the `typeSubTypeItemAssociations` endpoint doesn't have  a `lastUpdated` field. We need to know when records were last updated in order to perform a partial sync. So if the `lastUpdated` field is not present we can only ask the API for all records.

Reply:
Can I ask one thing here?

Work_role also doesn't have `lastUpdated` field. I don't look into that deeply, but doesn't this time info be used to compare the info in our sync log job table?

![image](https://user-images.githubusercontent.com/73486314/123719951-acde9000-d837-11eb-87d9-65e9d99ce655.png)

Reply:
I did expect `lastUpdated` to be available from "info" but it is just not there.
![image](https://user-images.githubusercontent.com/44041923/123870767-ac9dcd80-d8e7-11eb-9782-da65b74c981b.png)

My guess is that since these records can't be changed directly by users the updatedBy and lastUpdated fields are not available. 

This is what I got when trying to query with lastUpdated.
![image](https://user-images.githubusercontent.com/44041923/123871214-49f90180-d8e8-11eb-951f-0af8c85cf4c1.png)

Reply:
I got it. Thanks for the clarifications.